nvflare/job_config/fed_job_config.py

```diff
@@ -174,6 +176,11 @@ def _copy_ext_scripts(self, custom_dir, ext_scripts):
                 module = "".join(relative_script.rsplit(".py", 1)).replace(os.sep, ".")
                 self._copy_source_file(custom_dir, module, script, dest_file)
 
+    def _copy_ext_dirs(self, custom_dir, app_config: BaseAppConfig):
+        for dir in app_config.ext_dirs:
+            dest = os.path.join(custom_dir, os.path.basename(dir))
```

Comment:
Not sure this would work for Flower's use case. Here a subdir is created within the custom_dir, but in the Flower case, the content of the dir should be copied to the custom_dir directly - no subdir. The custom_dir is where their toml file should be stored.

On the other hand, I can see that the user might call add_ext_dir multiple times.

Maybe we should add a flag in add_ext_dir to indicate whether the content of the dir should be placed into the custom_dir directly.

Reply:
Also related, `_copy_ext_scripts()` can place the script in a subdir as well depending on the script path. Perhaps we would want to add the same flag here for placing it at the custom_dir instead? (this would require modifying obj._task_script_path too for ScriptExecutors)

Reply:
Discussed with @yanchengnv, changed the behavior to copying flat to job "custom" folder.